pages/auth/signin/signin.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btn = document.querySelector('.signin_button');
+
+function checkForm () {
+  const emailRegex = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}/;
+  const passwordRegEx = /^[A-Za-z0-9]{8,20}$/;
+  
+  const isEmailValid = emailRegex.test(email.value);
+  const isPasswordValid = passwordRegEx.test(password.value);
+
+  if (isEmailValid && isPasswordValid) {
+    btn.classList.add('btn_abled');
+  } else {
+    btn.classList.remove('btn_abled');
+  }
+};
+
+function btnCheck(e) {
+  if (!btn.classList.contains('btn_abled')) {
+    e.preventDefault();
+  }
+};
+
+email.addEventListener('keyup', checkForm);
+password.addEventListener('keyup', checkForm);
```

Comment:
둘다 input이고 input이라고 기대할만한 이름이다보니 keyup 보다는 input 이벤트가 좋을거같아요. 

아, signup 쪽에서도 이미 적용하셨군요 👍 

change이벤트도 시도해보시고 둘의 차이를 알고가셔도 좋을거같습니다

```suggestion
emailInput.addEventListener('input', checkForm);
passwordInput.addEventListener('input', checkForm);
```